### PR TITLE
Updated content for adherence to style guide and updated the older li…

### PR DESCRIPTION
**Pull request by Rajakavitha1**
…nks: /docs/user-guide/kubectl

The section about "Multiple Containers" read that they are examples that are work in progress. Can someone please confirm if we have any updates yet.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.10 Features: set Milestone to 1.10 and Base Branch to release-1.10
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
Updated the content to address the issue: https://github.com/kubernetes/website/issues/7686

File: docs/user-guide/walkthrough/index.md [hunk 1]
```diff
@@ -11,36 +11,36 @@ For Kubernetes 101, we will cover kubectl, pods, volumes, and multiple container
...
 {:toc}
 
 
 ## Kubectl CLI
 
-The easiest way to interact with Kubernetes is via the [kubectl](/docs/user-guide/kubectl-overview/) command-line interface.
+The easiest way to interact with Kubernetes is through the [kubectl](/docs/reference/kubectl/overview/) command-line interface.
 
-For more info about kubectl, including its usage, commands, and parameters, see the [kubectl CLI reference](/docs/user-guide/kubectl-overview/).
+For more info about kubectl, including its usage, commands, and parameters, see [Overview of kubectl](/docs/reference/kubectl/overview/).
```
Comment (steveperry-53) [inline docs/user-guide/walkthrough/index.md]: Probably don't need this sentence. It just repeats the link from the previous sentence.

**Pull request merged**